src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,23 @@
+.card__review-stars {
+  align-items: center;
+  display: flex;
+
+  .card__review-star {
+    display: flex;
+    background-image: url(../images/star.svg);
```

Comment:
Ensure that the path '../images/star.svg' is correct and that the image file exists at this location. Otherwise, the background image will not display.